### DISCUSSION TITLE
support mermaid for gatsby-starter-minimal-blog

### DISCUSSION THREAD

**Discussion by fmowl10** [LekoArts/gatsby-themes#1296] [Questions]
I want to render a mermaid.js diagram in a post on gatsby-starter-minimal-blog, so I followed [gatsby-remark-mermaid](https://www.gatsbyjs.com/plugins/gatsby-remark-mermaid/)'s guide, but it won't render.
How can I fix it?

**Comment by melloware** [comment]
I am trying to do the same and it is not working even though i am following the instructions: https://github.com/LekoArts/gatsby-themes/discussions/1143
